### PR TITLE
refactor: ab#209 extract general styles to general style sheet

### PR DESCRIPTION
**Pull request by DerCed**
## Description
Moved/Extracted/Rewritten styles

## Motivation and Context
Duplicate code

## How has this been tested?
Go through application

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by DerCed** [comment]
@witchpou Ich würde das gerne mit dir nochmal besprechen wollen - also WIP!

**Comment by witchpou** [comment]
> @witchpou Ich würde das gerne mit dir nochmal besprechen wollen - also WIP!

machen wir

File: webclient/app/src/app/features/apps/entities/entityCard/EntityCard.jsx [hunk 1]
```diff
@@ -36,9 +36,7 @@ function EntityCard(props) {
     function renderFieldsTable(entity) {
         if (!entity.fields || entity.fields.length === 0) {
             return (
-                <div className={entityCardStyles.statementWrapper}>
-                    <Statement message={t("entity.fields.empty")}/>
-                </div>
+                <Statement message={t("entity.fields.empty")}/>
```
Comment (witchpou) [inline webclient/app/src/app/features/apps/entities/entityCard/EntityCard.jsx:39]: das sieht aber so schon besser aus...

**Review by witchpou — CHANGES_REQUESTED** [review]

File: webclient/app/src/app/commons/statement/Statement.jsx [hunk 2]
```diff
@@ -5,8 +5,9 @@ import PropTypes from "prop-types";
 import iconEmpty from "./icon-empty.png";
 
 function Statement(props) {
-    const statementStyles = StatementStyles();
-    const {icon, message, actionMessage, onActionClick} = props;
+    const {icon, message, actionMessage, onActionClick, enableSpacing, spacingHeight} = props;
+
+    const statementStyles = StatementStyles({height: (enableSpacing && spacingHeight ? spacingHeight : "100%")});
```
[the remark this answers is not in the document]
Comment (DerCed) [reply webclient/app/src/app/commons/statement/Statement.jsx:10]: Bezogen auf die height?

[the remark this answers is not in the document]
Comment (DerCed) [reply webclient/app/src/app/commons/statement/Statement.jsx:10]: Die Abfrage noch in die Styling Komponente laden

**Pull request merged**